### ISSUE TITLE
Reduce the total amount of time spent on waiting on tests for Pycyphal when making pull requests

### ISSUE THREAD

**Issue by silverv** [OpenCyphal/pycyphal#239]
The branch will get tested in AppVeyor and the pull request will also get tested there, that's unnecessary double redundancy. The only argument for this is that it's easier to allocate time for something else in between.

**Comment by pavel-kirienko** [comment]
Superseded by https://github.com/OpenCyphal/pycyphal/issues/259
